package.json

```
@@ -27,18 +27,18 @@
 	"homepage": "https://github.com/ckeditor/ckeditor4-integrations-common#readme",
 	"browserslist": "> 0.25%, not dead",
 	"devDependencies": {
-		"@babel/core": "^7.11.6",
-		"@babel/preset-env": "^7.14.1",
+		"@babel/core": "^7.17.10",
```

```suggestion
		"@babel/core": "^7.18.2",
```
There is a newer version of `babel/core`